packages/angular_devkit/build_angular/src/utils/process-bundle.ts

```diff
@@ -97,7 +97,7 @@ export function setup(data: number[] | { cachePath: string; i18n: I18nOptions })
 
 async function cachePut(content: string, key: string | undefined, integrity?: string): Promise<void> {
   if (cachePath && key) {
-    await cacache.put(cachePath, key || null, content, {
+    await cacache.put(cachePath, key || null as unknown as string, content, {
```

Comment:
why is this cast needed?

Reply:
Because `null` is not accepted by the types provided via @types.

Reply:
We've already asserted that `key` is truthy on the line above, do we still need `null`?

Reply:
Dang you are right.